src/react/ui-elements/Veeam/VeeamCapacityModal.test.tsx

```diff
@@ -52,6 +53,16 @@ describe('VeeamCapacityModal', () => {
     expect(selectors.modalTitle()).toBeInTheDocument();
   });
 
+  it('should enabled edit button when value is valid', async () => {
+    await userEvent.click(selectors.editBtn());
+
+    await userEvent.clear(selectors.capacityInput());
+    expect(selectors.editModalBtn()).toBeDisabled();
+    await userEvent.type(selectors.capacityInput(), '2.2');
+
+    expect(selectors.editModalBtn()).toBeEnabled();
+  });
```

Comment:
nit: aren't we missing a test to check that numbers with more than 2 decimals are invalid ?

Reply:
There is a test for this:
`  it('should validate capacity value correctly : number with more than 2 decimals', async () => `